ansible/roles/postgresql/templates/postgresql.conf.j2

```diff
@@ -61,7 +61,7 @@ effective_cache_size = {{ postgresql_effective_cache_size }}
 log_destination = 'csvlog'
 logging_collector = on
 log_directory = 'pg_log'
-log_filename = 'postgresql-%x-%r.log'
+log_filename = 'postgresql-%Y-%m-%d_%H%M%S.log'
```

Comment:
This is the default. specified https://www.postgresql.org/docs/9.1/static/runtime-config-logging.html#RUNTIME-CONFIG-LOGGING-WHERE